### PR TITLE
fix(Viewer): Initial support for per-component lut/pwf

### PR DESCRIPTION
**Pull request by scottwittenburg**
Adds initial support for multi-component slice rendering.

Currently requires this [PR](https://github.com/Kitware/vtk-js/pull/1326) in order to try it out.

### CODE REVIEW

File: src/createViewer.js [hunk 1]
```diff
@@ -195,13 +198,21 @@ const createViewer = (
...
-          ].getLookupTable()
-          const sliceActors = store.imageUI.representationProxy.getActors()
-          sliceActors.forEach(actor => {
-            actor.getProperty().setRGBTransferFunction(lut)
-          })
+          const sliceActors = store.imageUI.representationProxy.getActors();
+          const dataArray = store.imageUI.image.getPointData().getScalars();
+          sliceActors.forEach((actor) => {
+            const actorProp = actor.getProperty();
+            actorProp.setIndependentComponents(1);
```
Comment (thewtex) [inline src/createViewer.js]: Here and vtk.js -- should this be a boolean?

**Comment by thewtex** [comment]
Looking great! :+1:

**Comment by oeway** [comment]
This is great! As I understand, this PR will also address https://github.com/Kitware/itk-vtk-viewer/issues/215 I posted before, right?

File: src/createViewer.js (hunk 1 above)
Comment (scottwittenburg) [reply src/createViewer.js]: Yes, I think that makes more sense.  Will do.

**Comment by thewtex** [comment]
@oeway yes! :champagne:

File: src/createViewer.js (hunk 1 above)
Comment (scottwittenburg) [reply src/createViewer.js]: Done in latest push to `vtk.js` PR...

**Comment by scottwittenburg** [comment]
@thewtex Final step here will be updating the version of `vtk.js` picked by the viewer (once that PR is approved and merged) and making sure the tests all pass at that point.

**Comment by thewtex** [comment]
@scottwittenburg looks beautiful:

MultiComponentVolumeRenderingOptimized

Bump vtk.js for merge?

**Comment by scottwittenburg** [comment]
@thewtex We should be all set already, I bumped this morning when I noticed you had merged the `vtk.js` topic.

File: src/createViewer.js [hunk 2]
```diff
@@ -189,18 +189,23 @@ const createViewer = (
...
           const sliceActors = store.imageUI.representationProxy.getActors()
           sliceActors.forEach(actor => {
-            actor.getProperty().setRGBTransferFunction(lut)
+            const actorProp = actor.getProperty();
+            actorProp.setIndependentComponents(true);
+            for (let component = 0; component < numberOfComponents; component++) {
+              const lutProxy = store.imageUI.lookupTableProxies[component];
+              const pwfProxy = store.imageUI.piecewiseFunctionProxies[component];
+              actorProp.setRGBTransferFunction(component, lutProxy.getLookupTable());
+              actorProp.setScalarOpacity(component, pwfProxy.getPiecewiseFunction());
```
Comment (thewtex) [inline src/createViewer.js]: @scottwittenburg `setScalarOpacity` -> `setPiecewiseFunction`, correct?

Comment (scottwittenburg) [reply src/createViewer.js]: Whoops, oversight on my part.  Will fix.

**Pull request merged**